autogen.sh

```diff
@@ -17,3 +17,5 @@ fi
 aclocal
 autoconf
 automake --add-missing
+
+echo "now you can run ./configure && make to build openfortivpn" >/dev/stderr
```

Comment:
In Bourne shell, shouldn't we use `>&2` instead of `>/dev/stderr`?

I believe `>/dev/stderr` is **not** portable, and wasn't even aware it can work on Linux...

Reply:
same here, I have changed it to `>&2` as requested